pkg/controller/controller_utils_test.go

```diff
@@ -601,29 +702,41 @@ func TestSortingActivePodsWithRanks(t *testing.T) {
 			if podsWithRanks.Less(1, 0) {
 				t.Errorf("expected pod %q with rank %v not to be less than %v with rank %v", podsWithRanks.Pods[1].Name, podsWithRanks.Rank[1], podsWithRanks.Pods[0].Name, podsWithRanks.Rank[0])
 			}
-		})
-	}
+		}
+	})
 }
 
 func TestActiveReplicaSetsFiltering(t *testing.T) {
-	var replicaSets []*apps.ReplicaSet
-	replicaSets = append(replicaSets, newReplicaSet("zero", 0))
-	replicaSets = append(replicaSets, nil)
-	replicaSets = append(replicaSets, newReplicaSet("foo", 1))
-	replicaSets = append(replicaSets, newReplicaSet("bar", 2))
-	expectedNames := sets.NewString()
-	for _, rs := range replicaSets[2:] {
-		expectedNames.Insert(rs.Name)
-	}
 
-	got := FilterActiveReplicaSets(replicaSets)
-	gotNames := sets.NewString()
-	for _, rs := range got {
-		gotNames.Insert(rs.Name)
+	rsUuid := uuid.NewUUID()
+	tests := []struct {
+		name                string
+		replicaSets         []*apps.ReplicaSet
+		expectedReplicaSets []*apps.ReplicaSet
```

Comment:
```suggestion
		wantReplicaSets []*apps.ReplicaSet
```
For consistency.

Reply:
This is not applied

Reply:
sorry, I missed it for some reason, fixed